homeassistant/components/camera/proxy.py

```diff
@@ -136,7 +137,8 @@ def __init__(self, hass, config):
         self._cache_images = bool(
             config.get(CONF_IMAGE_REFRESH_RATE)
             or config.get(CONF_CACHE_IMAGES))
-        self._last_image_time = 0
+        self._last_image_time = dt.datetime.utcfromtimestamp(
+            0).replace(tzinfo=dt_util.UTC)
```

Comment:
`util/dt.py` is our helper to deal with time-related stuff.

Reply:
I didn't see anything there that seemed useful; what abstraction would you like me to add to be able to set a default timestamp (tz aware)?

Reply:
try:
self._last_image_time = dt_util.utc_from_timestamp(0)

Otherwise, looks ok to me

Reply:
huh... not quite sure how I missed that.